server/api/actions/migrations.js

```diff
@@ -0,0 +1,29 @@
+const util = require('util');
+const fs = require('fs');
+
+/**
+ * @description Read all files in current directory, filter out files
+ * that don't contain SQL and previously ran migrations.
+ * Read content of those files.
+ * @param {array} migrations 
+ */
+
+const getOutStandingMigrations = async (migrations = []) => {
+  const files = await util.promisify(fs.readdir)('server/sql/migrations');
+  const sql = await Promise.all(
+    files
+      .filter((file) => file.split('.')[1] === 'sql')
+      .filter((file) => !migrations.includes(file))
```

Comment:
you should be able to just combine these

```suggestion
      .filter((file) => file.split('.')[1] === 'sql' &&  !migrations.includes(file))
```